comet/comet.go

```diff
@@ -70,7 +70,7 @@ func (s *ABCI) Info(ctx *jsonrpctypes.Context) (*rpctypes.ResultABCIInfo, error)
 }
 
 type HeadBlocker interface {
-	HeadBlock() *monomer.Block
+	HeadHeader() (*monomer.Header, error)
```

Comment:
Should the `HeadBlocker` interface be renamed now that it only returns the header?